src/applications/vaos/new-appointment/components/DateTimeRequestOptions.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import classNames from 'classnames';
+
+/*
+ * Simiarly to slots, but for checkboxes, which we know we only ever have two of
+ * So calculations are against either the first or the second one
+ */
+function getCheckboxOptionClasses(index) {
+  return classNames('vaos-calendar__option-cell', {
+    'vaos-u-border-radius--top-left': index === 0,
+    'vaos-u-border-radius--top-right': index === 1,
+    'vaos-u-border-radius--bottom-left': index === 0,
+    'vaos-u-border-radius--bottom-right': index === 1,
+    'vads-u-padding-left--2': index === 0,
+    'vads-u-padding-top--2': true,
+    'vads-u-padding-right--2': index === 1,
+  });
+}
+
+export default function DateTimeRequestOptions({
```

Comment:
I think it makes sense to create a folder for DateTimeRequestPage and put this component in it, but that seemed like a change that could wait for a separate PR.